src/App.js

```diff
@@ -1,6 +1,6 @@
 import { Route,Routes } from "react-router-dom";
 import "./App.css";
-import {Home,History,WatchLater,Liked,ErrorPage,Videolist, LoginPage } from "./pages/index";
+import {Home,History,WatchLater,Liked,ErrorPage,Videolist, LoginPage, VideoPage } from "./pages/index";
 
```

Comment:
no need of  pages/index, directly can do pages as it  by default imports from index

Reply:
Hey Raman, in index im _exporting_ all page components so I gave pages/index